lib/workers/repository/update/branch/get-updated.ts

```diff
@@ -40,6 +40,35 @@ async function getFileContent(
   return fileContent;
 }
 
+function sortPackageFiles(
+  config: BranchConfig,
+  manager: string,
+  packageFiles: FileAddition[],
```

Comment:
```suggestion
  packageFiles: FilePath[],
```

you can also make it generic if you like

Reply:
Couldn't find the `FilePath`, what is it? A type alias for string meant to be created?

Reply:
needs:
- https://github.com/renovatebot/renovate/pull/30725

Reply:
Done